css/styles.css

```diff
@@ -19,18 +19,7 @@
   display: none;
 }
 
-.delete-modal {
-  position: fixed;
-  top: 0;
-  left: 0;
-  z-index: 1060;
-  width: 100%;
-  height: 100%;
-  overflow: hidden;
-  outline: 0;
-}
-
-.favorite-modal {
+.my-modal {
```

Comment:
We typically want to avoid using `my` in variable and class names. You can use something like just `modal` or if that is used for something else you can do something like `brewery-modal`